internal/policies/mount/adsys_mount/src/main.rs

```diff
@@ -0,0 +1,235 @@
+use clap::Parser;
+use gio::{
+    self,
+    traits::{FileExt, MountOperationExt},
+};
+use glib::ObjectExt;
```

Comment:
is it used anywhere?

Reply:
Yes... There are some "methods" that require traits to be imported